pydoover/config/__init__.py

```diff
@@ -188,13 +188,15 @@ def __init__(
         description: str = None,
         deprecated: bool = None,
         hidden: bool = False,
+        format: str = None,
```

Comment:
yeah maybe if you merge pydoover back in it might have some merge conflicts for this but all disappear these few lines